File: app/src/main/java/com/summer/itis/summerproject/model/AbstractCard.java
package com.summer.itis.summerproject.model;

public class AbstractCard {

    private String id;

    private String name;

    private String photoUrl;

    private String extract;

    private String description;

    private String wikiUrl;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPhotoUrl() {
        return photoUrl;
    }

    public void setPhotoUrl(String photoUrl) {
        this.photoUrl = photoUrl;
    }

    public String getExtract() {
        return extract;
    }

    public void setExtract(String extract) {
        this.extract = extract;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getWikiUrl() {
        return wikiUrl;
    }

    public void setWikiUrl(String wikiUrl) {
        this.wikiUrl = wikiUrl;
    }
}

File: app/src/main/java/com/summer/itis/summerproject/model/AbstractCard.kt
package com.summer.itis.summerproject.model

import com.google.firebase.database.Exclude
import com.google.firebase.database.IgnoreExtraProperties
import com.google.gson.annotations.Expose

@IgnoreExtraProperties
class AbstractCard {

    var id: String? = null

    var name: String? = null

    var photoUrl: String? = null

    var extract: String? = null

    var description: String? = null

    var wikiUrl: String? = null

    @field:Exclude var isOwner: Boolean = false
}
